src/components/global/TaskList.jsx

```diff
@@ -14,25 +15,35 @@ function TaskList(props) {
   const [currentTask] = useContext(CurrentTaskContext);
   const { isDarkMode } = React.useContext(DarkModeContext);
 
+  const today = new Date().toDateString();
+  const seconds = moment(today, false).toDate().getTime();
+
+  const checkDate = (startTime, endTime) =>
+    seconds <= endTime * 1000 && seconds >= startTime * 1000;
+
   return (
     <div className={isDarkMode ? "task-list" : "task-list light"}>
       <h1 className="task-list__title">Today&apos;s Tasks:</h1>
       <div className="task-list__content">
         <div className="task-list__tasks">
           {tasks.map((t) => (
             <div key={t.Task_id}>
-              <Task
-                name={t.Name}
-                onClick={() => onTaskClick(t)}
-                selected={currentTask && t.Task_id === currentTask.Task_id}
-                endDate={t.End_date}
-                label={t.Label}
-                startDate={t.Start_date}
-                taskId={t.Task_id}
-                checked={t.Is_complete}
-                description={t.Description}
-                userId={t.User_id}
-              />
+              {t.Start_date &&
```

Comment:
Currently you are able to successfully create a task without an end or start date. This new logic will mean that even though that tasks gets persisted in the database, it won't show in the task list. This is not necessarily a bad thing, just something to consider.

Reply:
Nice catch, was an easy fix, Thanks Denise :)